src/Form/CaseFilterType.php

```diff
@@ -238,5 +244,33 @@ function (FormEvent $event) use ($formModifier) {
                 ])
             ;
         }
+
+        $builder->add('activeFilter', Filters\ChoiceFilterType::class, [
+            'choices' => [
+                $this->translator->trans('Active', [], 'case') => CaseSpecialFilterStatuses::ACTIVE,
+                $this->translator->trans('Not active', [], 'case') => CaseSpecialFilterStatuses::NOT_ACTIVE,
+            ],
+            'apply_filter' => function (QueryInterface $filterQuery, $field, $values) {
+                if (empty($values['value'])) {
+                    return null;
+                }
+
+                $filterChoice = $values['value'];
+
+                // Modify query builder according to filter choice
+                switch ($filterChoice) {
+                    case CaseSpecialFilterStatuses::ACTIVE:
+                        $filterQuery->getQueryBuilder()->andWhere($this->caseEntityRepository->getExprWithBoardFinishStatuses($filterQuery->getQueryBuilder()));
+                        break;
+                    case CaseSpecialFilterStatuses::NOT_ACTIVE:
+                        $filterQuery->getQueryBuilder()->andWhere($this->caseEntityRepository->getExprWithBoardFinishStatuses($filterQuery->getQueryBuilder(), false));
+                        break;
+                }
```

Comment:
Can/should be written as 
```
$filterQuery->getQueryBuilder()->andWhere($this->caseEntityRepository->getExprWithBoardFinishStatuses($filterQuery->getQueryBuilder(), CaseSpecialFilterStatuses::ACTIVE === $filterChoice));
```

Why do you modify the query builder rather than add conditions on `$filterQuery`?

Reply:
Added conditions on `$filterQuery` instead in b30daf0c98cdcedb79836d90b6e3907cc36cfaf6.

It looks to have the same effect, but adding conditions as you mention seem to be the correct way (https://github.com/lexik/LexikFormFilterBundle/blob/master/Resources/doc/working-with-the-bundle.md#ii-filter-customization)